Automattic/Gravatar-SDK-iOS
# Adding tutorial to docs

Closes #

### Description

This PR adds a tutorial on how to use Profile Views as UITableView cells.

The intention is to try the swift-docc tutorial mechanism and decide if it's something we want to use.

### Testing Steps

- Product -> Build Documentation
- See the Tutorials bundle:
![CleanShot 2024-05-31 at 11 50 47@2x](https://github.com/Automattic/Gravatar-SDK-iOS/assets/9772967/e55bef53-9b56-4034-a467-0b4d7de68795)
- Open the GravatarUI Docs bundle
- See the new Tutorials section:
![CleanShot 2024-05-31 at 11 51 47@2x](https://github.com/Automattic/Gravatar-SDK-iOS/assets/9772967/df06fbc0-7352-4b2b-bd9e-774efc546d8b)
- Follow the tutorial.
- Check if the solution implementation is correct. (The idea is a minimal workable solution which can be expanded on)
